src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -248,8 +245,7 @@ public void Abort()
                         ResponseBody = _responseBody;
                         DuplexStream = new FrameDuplexStream(RequestBody, ResponseBody);
 
-                        _requestAbortCts = CancellationTokenSource.CreateLinkedTokenSource(_disconnectCts.Token);
-                        RequestAborted = _requestAbortCts.Token;
+                        RequestAborted = _disconnectOrAbortedCts.Token;
```

Comment:
Currently the behaviour is after the current request finishes it triggers the abort token; and if you actually call `Abort()` it aborts and disconnects?

Will change to maintain that behaviour